README.md

```diff
@@ -21,7 +21,13 @@ To get a local copy up and running follow these simple example steps.
 👤 **Farouq Daif**
 
 - GitHub: [@farouqdaif1](https://github.com/farouqdaif1)
-- LinkedIn: [@Farouq Daif ](https://www.linkedin.com/in/farouqdaif/)
+- LinkedIn: [@Farouq Daif ](https://www.linkedin.com/in/farouqdaif/)\
+- 
+  ## 👤Author
+
+- GitHub: [@butlermuwo](https://github.com/butlermuwo)
+- Twitter: [@ButlerMuwo](https://twitter.com/ButlerMuwo)
+- LinkedIn: [butlermuwo](https://www.linkedin.com/in/butlermuwo/)
```

Comment:
- [ ] (Optional)In your `README.md` file, you can include the command to run the console app after they have cloned it. That would be adding `run ruby main.rb` to start the console app and interact with it. 👍

Reply:
Thanks for Feedback 😄  I will change it 👍